spack/README.md

```diff
@@ -10,38 +10,56 @@ The workflow `build_spack` in `.github/workflows/build-spack.yml` uses the image
 
 ## When a new Spack release is necessary
 
-* Add checksum of newest version(s) to https://github.com/precice/python-bindings/blob/develop/spack/repo/packages/py-pyprecice/package.py. You can get checksum for any released version by running `spack checksum py-pyprecice`.
+* Add checksum of newest version(s) to the [package recipe in `package.py`](https://github.com/precice/python-bindings/blob/develop/spack/repo/packages/py-pyprecice/package.py). You can get checksum for any released version by running `spack checksum py-pyprecice`.
 * Check whether the new release works locally. You can add the Spack repo in this repository and do the following:
+
   1. Add the repository to your Spack installation if you have not done that before. You can check the repositories available with your Spack installation via `spack repo list`. If there is a repository called `pyprecice.test`, check whether it points to the correct path. The correct path is the `spack/repo` directory of python-bindings repository. If the repository is missing, add it via
-  ```bash
-  spack repo add ${REPOSITORY_ROOT}/spack/repo
-  ```
+
+     ```bash
+     spack repo add ${REPOSITORY_ROOT}/spack/repo
+     ```
+
   2. Build the new python bindings. Usually, we use a new Spack environment. This is created and loaded via
-  ```
-  spack env create pyprecicetest-VERSIONNUMBER
-  spack env activate -p pyprecicetest-VERSIONNUMBER
-  spack
-  ```
-  Note that the dots `.` in the version number must be replaced by underscores or similar.
+
+     ```text
+     spack env create pyprecicetest-VERSIONNUMBER
+     spack env activate -p pyprecicetest-VERSIONNUMBER
+     ```
+
+     Note that the dots `.` in the version number must be replaced by underscores or similar. This means, one would create an environment named `pyprecicetest-2_0_0_1` instead of `pyprecicetest-2.0.0.1` since the `.` character is not allowed for Spack environments.
+
   3. We add the `py-pyprecice` package from our repository to the environment via
-  ```
-  spack add pyprecice.test.py-pyprecice@VERSIONNUMBER
-  ```
-  This ensures that one uses the local repository which has the namespace `pyprecice.test` when installing `py-precice`.
+
+     ```text
+     spack add pyprecice.test.py-pyprecice@VERSIONNUMBER
+     ```
+
+     This ensures that one uses the local repository which has the namespace `pyprecice.test` when installing `py-precice`. Note that when speciying the `VERSIONNUMBER` you use the actual version number with dots, e.g. `2.0.0.1`.
```

Comment:
Is the user not required to mention the full version as `v2.0.0.1`, or is just `2.0.0.1` enough? I have not tried this myself.